### ISSUE TITLE
Download shared splitted file as 1 file?

### ISSUE THREAD

**Issue by rromm** [yar229/WebDavMailRuCloud#165]
heard that WDMRC splitting big files cause of size limits.
but I'm almost sure there's no mentioned function (otherwise where's the info about splitting placed).

**Comment by yar229** [comment]
Yep, big files splits to chunks by cloud filesize limit detected automatically (usually it's 2Gb for free accounts). 

That's how you can see one and work with in your file manager:

![image](https://user-images.githubusercontent.com/5150160/67268647-abcf5a00-f4bd-11e9-9dc2-4388d70a064a.png)

And splitted files in cloud:

![image](https://user-images.githubusercontent.com/5150160/67268769-fbae2100-f4bd-11e9-931f-cf29abf3b0b4.png)

video3.mkv really is a header file
video3.mkv.wdmrc.001 and video3.mkv.wdmrc.002 contains data.

**Comment by rromm** [comment]
ok but what about subject? if I share those 3 files will friend see 1 file just like me in his WDMRC?

**Comment by rromm** [comment]
by the way I wouldn't change format of data keeping it video, because it's usually watchable on web cloud mail ru.

**Comment by rromm** [comment]
at least some players could open it so I could share those couple of data for those who doesn't have WDMRC.

**Comment by yar229** [comment]
> ok but what about subject? if I share those 3 files will friend see 1 file just like me in his WDMRC?

1. If you share `video3.mkv`  using special command `>>share video3.mkv` you'll got 3 links (take a look on file `video3.mkv.share.wdmrc`), it's not the way you use to.

2. If you share a folder, your friend can attach it to his cloud using wdmrc spec.command `>>link` (`.,.link`) and then he can see `video3.mkv` as single file.

> by the way I wouldn't change format of data keeping it video, because it's usually watchable on web cloud mail ru.

There's a big trouble - Cloud web interface cannot play splitted videos.


> at least some players could open it so I could share those couple of data for those who doesn't have WDMRC.

Maybe so, but not for all video formats.

**Comment by rromm** [comment]
>− if I share those 3 files — will friend see 1 file just like me in his WDMRC?
>− If you share video3.mkv using special command >>share video3.mkv you'll got 3 links
I assume it's yes.
but for convinient sharing it would be good to place data in splitted (uncompressed?) archive, for example — zip opens as folder in windows, so files is accessible while it's in some (any) part of archive.
as for videos we could test splitted formats for playing, after all — such sharing is better than nothing (to those friends etc without WDMRC).
